src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_consts.py

```diff
@@ -19,7 +19,7 @@
 
 LOGIN_METHOD_LIST = [
     'Device code authentication, we will provide a code you enter into a web page and log into',
-    'Username and password',
+    "Username and password(won't work for MFA enforced accounts or MSA accounts such as live-id)",
```

Comment:
@derekbekoe, this is the message I updated per clarification towards a user inquiry